docs/metrics.md

```diff
@@ -20,163 +20,190 @@ They reflect and describe exactly what is being exposed.
+Storage read traffic in bytes. Type: Counter.
 
 ### kubevirt_vmi_storage_write_times_ms_total
-Storage write operation time.
+Storage write operation time. Type: Counter.
 
 ### kubevirt_vmi_storage_write_traffic_bytes_total
-Storage write traffic in bytes.
+Storage write traffic in bytes. Type: Counter.
 
 ### kubevirt_vmi_vcpu_seconds
-Amount of time spent in each state by each vcpu. Where `id` is the vcpu identifier and `state` can be one of the following: [`OFFLINE`, `RUNNING`, `BLOCKED`].
+Amount of time spent in each state by each vcpu. Where `id` is the vcpu identifier and `state` can be one of the following: [`OFFLINE`, `RUNNING`, `BLOCKED`]. Type: Counter.
 
 ### kubevirt_vmi_vcpu_wait_seconds
-Amount of time spent by each vcpu while waiting on I/O.
+Amount of time spent by each vcpu while waiting on I/O. Type: Counter.
 
 <div id='containerized-data-importer'></div>
 
 ## [containerized-data-importer](https://github.com/kubevirt/containerized-data-importer/tree/main)
 
+### clone_progress
+The clone progress in percentage. Type: Counter.
 ### kubevirt_cdi_clone_dv_unusual_restartcount_total
-Total restart count in CDI Data Volume cloner pod
+Total restart count in CDI Data Volume cloner pod. Type: Counter.
+### kubevirt_cdi_cr_ready
+CDI CR Ready. Type: Gauge.
+### kubevirt_cdi_dataimportcron_outdated
+DataImportCron has an outdated import. Type: Gauge.
 ### kubevirt_cdi_dataimportcron_outdated_total
-Total count of outdated DataImportCron imports
+Total count of outdated DataImportCron imports. Type: Counter.
 ### kubevirt_cdi_import_dv_unusual_restartcount_total
-Total restart count in CDI Data Volume importer pod
+Total restart count in CDI Data Volume importer pod. Type: Counter.
+### kubevirt_cdi_incomplete_storageprofiles_total
+Total number of incomplete and hence unusable StorageProfile. Type: Gauge.
 ### kubevirt_cdi_operator_up_total
-CDI operator status
+CDI operator status. Type: Gauge.
 ### kubevirt_cdi_upload_dv_unusual_restartcount_total
-Total restart count in CDI Data Volume upload server pod
+Total restart count in CDI Data Volume upload server pod. Type: Counter.
 <div id='cluster-network-addons-operator'></div>
 
 ## [cluster-network-addons-operator](https://github.com/kubevirt/cluster-network-addons-operator/tree/main)
 
 ### kubevirt_cnao_cr_kubemacpool_deployed
-KubeMacpool is deployed by CNAO CR
+KubeMacpool is deployed by CNAO CR. Type: Gauge.
+### kubevirt_cnao_cr_kubemacpool_deployed_total
+Total count of KubeMacPool manager pods deployed by CNAO CR. Type: Gauge.
 ### kubevirt_cnao_cr_ready
-CNAO CR Ready
+CNAO CR Ready. Type: Gauge.
+### kubevirt_cnao_kubemacpool_manager_num_up_pods_total
+Total count of running KubeMacPool manager pods. Type: Gauge.
+### kubevirt_cnao_num_up_operators
+Total count of running CNAO operators. Type: Gauge.
+### kubevirt_kubemacpool_duplicate_macs_total
+Total count of duplicate KubeMacPool MAC addresses. Type: Gauge.
 <div id='ssp-operator'></div>
```

Comment:
@aadmi @machadovilaca Hi, I see we have an issue here. Can you please check?

Reply:
Hi @sradco, are you referring to the metrics types? 
I think it was added in [#1330](https://github.com/kubevirt/cluster-network-addons-operator/pull/1330). 
@machadovilaca, are these the right metrics types?